Tests/TestingTests/EventRecorderTests.swift

```diff
@@ -467,15 +467,35 @@ struct EventRecorderTests {
   @Test("JUnitXMLRecorder counts issues without associated tests")
   func junitRecorderCountsIssuesWithoutTests() async throws {
     let issue = Issue(kind: .unconditional)
-    let event = Event(.issueRecorded(issue), testID: nil, testCaseID: nil)
     let context = Event.Context(test: nil, testCase: nil, configuration: nil)
 
-    let recorder = Event.JUnitXMLRecorder { string in
-      if string.contains("<testsuite") {
-        #expect(string.contains(#"failures=1"#))
```

Comment:
This is a drive-by fix to the pre-existing test above the new one I added below: This one wasn't actually working, since no `.runEnded` event was recorded, so this callback wasn't happening. Once I fixed that I noticed a secondary issue, which is that the `"failures=1"` string was missing double quotes around the number 1.